dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -32,12 +32,8 @@ local tputests = base.BaseTest {
       pip install -e .
       echo $KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS
       export XRT_TPU_CONFIG="tpu_worker;0;${KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS:7}"
-      coverage run --source=pytorch_lightning -m pytest -v --capture=no \
-          tests/profiler/test_xla_profiler.py \
-          pytorch_lightning/utilities/xla_device.py \
-          tests/accelerators/test_tpu.py \
-          tests/callbacks/test_device_stats_monitor.py \
-          tests/models/test_tpu.py
+      export PL_RUN_TPU_TESTS=1
+      coverage run --source=pytorch_lightning -m pytest -v --capture=no tests
```

Comment:
The standalone script does the grep to filter the list of files to collect because the collection for the whole test suite takes a bit.

I guess it's not a problem here because tpu tests are not the current bottleneck in our CI

Reply:
The difference is that we don't need to run each test with a separate pytest process call, since for TPU we always use spawn and can  therefore use a single pytest call.

Reply:
touché